.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: ipfs/ci-websites
+        environment:
+          DEBUG: "true"
+    steps:
+      - checkout
+      - run:
+          command: make install
+      - run:
+          command: make build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - public
+
+  deploy:
+    docker:
+      - image: olizilla/ipfs-dns-deploy:1.1
```

Comment:
@olizilla this seems neat and potentially super useful. Is the source code for it available anywhere? (It’s not linked from Docker Hub, unless I’m missing something.)

Reply:
@Mr0grog for sure! https://github.com/ipfs-shipyard/ipfs-dns-deploy

Reply:
Ooh, that’s even fancier than I’d realized! Nice 👌